composer.json

```diff
@@ -25,26 +29,26 @@
     }
   ],
   "require": {
-    "coreshop/resource-bundle": "^2.0 | ^3.0",
-    "coreshop/rule-bundle": "^2.0 | ^3.0",
-    "coreshop/pimcore-bundle": "^2.0 | ^3.0",
-    "coreshop/pimcore": "^2.0 | ^3.0",
-    "pimcore/pimcore": "^6.5",
-    "league/csv": "^9.0",
-    "box/spout": "^2.7"
+    "php": ">=8.0",
+    "ext-json": "*",
+    "box/spout": "^3.0",
```

Comment:
This dependency update (from `^2.7` to `^3.0`) doesn't go well with the Pimcore CMF (see https://github.com/pimcore/customer-data-framework/blob/master/composer.json#L9), but IMO they should update their dependency as well.

Reply:
There is already a pull request for the upcoming CMF Version 3.1

Reply:
Ah right, didn't notice that. Even better. 👍🏻